web/index.php

```diff
@@ -55,8 +62,16 @@
     $twilio->say(sprintf("The current on-call engineer is %s." .
         "%s Please hold while we connect you.",
         $user['first_name'], $time), $attributes);
-
-    $dial = $twilio->dial(NULL, array('action' => "check_if_completed_by_human.php", 'timeout' => 25));
+    
+    $dialvars = array('action' => "check_if_completed_by_human.php", 'timeout' => 25);
+    if ($record != strtolower("false")) {
+        if ($record == strtolower("true")) {
```

Comment:
Oops, missed that mistake; clearly I was moving too fast and got that
wrong. Thanks!

On Tue, May 31, 2016, 8:27 PM MDCollins <notifications@github.com> wrote:

> In web/index.php
> <https://github.com/mdcollins05/phoneduty/pull/1#discussion_r65285874>:
>
> > @@ -55,8 +62,16 @@
> >      $twilio->say(sprintf("The current on-call engineer is %s." .
> >          "%s Please hold while we connect you.",
> >          $user['first_name'], $time), $attributes);
> > -
> > -    $dial = $twilio->dial(NULL, array('action' => "check_if_completed_by_human.php", 'timeout' => 25));
> > +
> > +    $dialvars = array('action' => "check_if_completed_by_human.php", 'timeout' => 25);
> > +    if ($record != strtolower("false")) {
> > +        if ($record == strtolower("true")) {
>
> I made some small changes here because this didn't seem to do what I think
> it was expected to do. I used the following instead:
>
> if (strtolower($record) == 'true') {
>
> I made a similar change to the if statement above.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/mdcollins05/phoneduty/pull/1/files/468983823ddd67c474a52eb0340dc68a19616bd7#r65285874>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AALz4WC-TvJBJG_PWedbjF0CIHGdDuG4ks5qHNIJgaJpZM4G9c3S>
> .
>
-- 

John G. Gauthier